pkg/config/validation/validation.go

```diff
@@ -1487,19 +1487,25 @@ func ValidateAuthenticationPolicy(name, namespace string, msg proto.Message) err
 }
 
 func validateWorkloadSelector(selector *type_beta.WorkloadSelector) error {
+	var errs error
 	if selector != nil {
 		for k, v := range selector.MatchLabels {
-			if k == "" || v == "" {
-				return fmt.Errorf("selector has empty key or values")
+			if k == "" {
```

Comment:
Do we have use case that allows empty values?

Reply:
I think it depends on user, I removed to allow users to set empty value in selector mainly because k8s allows label to have empty value.

Reply:
Are the two cases equivalent? Having a label set to empty, or not having the label set.

Reply:
It's different. A label with empty value is not special and no different from a normal label with non-empty value. But without the label is another thing.